src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DayRent = 40;
+  let totalRent = 0;
+
+  totalRent = DayRent * days;
+
+  if (days >= 7) {
+    totalRent = DayRent * days - 50;
```

Comment:
Use a constant for the `7` and `50` to avoid hardcoding values directly in the expression. For example, `const LONG_TERM_RENTAL_DAYS = 7;` and `const LONG_TERM_DISCOUNT = 50;`.